x/ccv/provider/keeper/relay.go

```diff
@@ -30,8 +30,7 @@ func (k Keeper) OnAcknowledgementPacket(ctx sdk.Context, packet channeltypes.Pac
 			"error", err,
 		)
 		if consumerId, ok := k.GetChannelIdToConsumerId(ctx, packet.SourceChannel); ok {
-			// stop consumer chain and release unbonding
-			return k.StopConsumerChain(ctx, consumerId, false)
```

Comment:
If you try to close the channel here it will result in an error as the channel will already be closed in the IBC module. That's the reason we had the `closeChan = false` argument. What's the reason for removing it?

Reply:
Discussed offline. There's no `panic` when trying to close the channel, so there's no need for `closeChan` any longer.